lib/icalendar/alarm.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ICalendar.Alarm do
+  @moduledoc """
+  Events have alarms.
+  """
+
+  defstruct minutes_before: 60
```

Comment:
I think that rather than specifying an `Int` of minutes we should pass in some generic type that represents a positive relative amount of time.

Perhaps a time tuple? These are popular in the Erlang world.

```elixir
{mega_seconds, seconds, micro_seconds}
```

Or maybe something with the new time types in Elixir 1.4.